seqr/views/apis/variant_search_api.py

```diff
@@ -534,19 +534,78 @@ def _flatten_variants(variants):
 @login_and_policies_required
 def variant_lookup_handler(request):
     kwargs = {_to_snake_case(k): v for k, v in request.GET.items()}
-    include_genotypes = kwargs.pop('include_genotypes', False)
-    families = None
-    if include_genotypes:
+
+    variant_id = kwargs.pop('variant_id')
+    parsed_variant_id = parse_variant_id(variant_id)
+    is_sv = not parsed_variant_id
+    if is_sv:
         families = _all_genome_version_families(
             kwargs.get('genome_version', GENOME_VERSION_GRCh38), request.user,
         )
+        if not families:
+            raise PermissionDenied()
+        variants = sv_variant_lookup(request.user, variant_id, families, **kwargs)
+    else:
+        variant = variant_lookup(request.user, parsed_variant_id, **kwargs)
+        variants = [variant]
+        families = Family.objects.filter(
+            guid__in=variant['familyGenotypes'],
+            project__guid__in=get_project_guids_user_can_view(request.user, limit_data_manager=True),
+        )
+        variant['familyGuids'] = list(families.values_list('guid', flat=True))
 
-    variants = variant_lookup(request.user, families=families, **kwargs)
-    saved_variants, _ = _get_saved_variant_models(variants, families) if families else (None, None)
+    saved_variants, _ = _get_saved_variant_models(variants, None) if families else (None, None)
     response = get_variants_response(
         request, saved_variants=saved_variants, response_variants=variants,
-        add_all_context=include_genotypes, add_locus_list_detail=include_genotypes,
+        add_all_context=True, add_locus_list_detail=True,
     )
     response['variants'] = variants
 
+    if not is_sv:
+        _update_lookup_variant(variant, response)
+
     return create_json_response(response)
+
+
+def _update_lookup_variant(variant, response):
+    individual_guid_map = {
+        (i['familyGuid'], i['individualId']): i['individualGuid'] for i in response['individualsByGuid'].values()
+    }
+
+    no_access_families = set(variant['familyGenotypes']) - set(variant['familyGuids'])
```

Comment:
what's the difference between what `variant['familyGenotypes']` and `variant['familyGuids']` are representing?